frontend/src/v5/ui/components/shared/modals/templates/deleteModal/deleteModal.component.tsx

```diff
@@ -37,9 +37,19 @@ interface IDeleteModal {
 	name: string,
 	message?: string,
 	confidenceCheck?: boolean,
+	titleLabel?: string,
+	confirmLabel?: string,
 }
 
-export const DeleteModal = ({ onClickConfirm, onClickClose, name, message, confidenceCheck }: IDeleteModal) => {
+export const DeleteModal = ({
+	onClickConfirm,
+	onClickClose,
+	name,
+	message,
+	confidenceCheck,
+	titleLabel,
+	confirmLabel,
```

Comment:
I dont get this in the component is being used as boolean only.

Reply:
confirmLabel and titleLabel aren't being used as boolean. If they are provided then they are used. If they are not provided then there is a default that takes over